### DISCUSSION TITLE
canva pro version

### DISCUSSION THREAD

**Discussion by shinenawo** [community/community#29620] [GitHub Education]
hi,
may i know why i can't get canva pro version with github as a university student. please help me.

**Comment by melvincwng** [comment]
@shinenawo , Hi there, Canva Pro is no longer available in the GitHub Student Developer Pack as of now.

Hope this clarifies 😊

**Reply by melvincwng** [reply]
FYI, it used to be available.

But as of now, Canva Pro seems to have been removed from the list of benefits in the GitHub Student Developer Pack.

You can refer to this link for more information on what benefits are available still: https://education.github.com/pack

Hope this helps
